runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -58,6 +60,49 @@ public static void renderPolygon(Graphics2D graphics, Polygon poly, Color color)
 		graphics.setStroke(originalStroke);
 	}
 
+	public static void renderActorList(Graphics2D graphics, Client client, List<NPC> actorList, Color color, OverlayStyle style, boolean drawNames)
```

Comment:
Well by common method I meant that both slayer and NPC indicators plugins should use this method